src/linearmixedmodel.jl

```diff
@@ -358,7 +350,7 @@ function condVartables(m::MixedModel{T}) where {T}
     return NamedTuple{fnames(m)}((map(_cvtbl, condVar(m), m.reterms)...,))
 end
 
-function pushALblock!(A, L, blk)
+function _pushALblock!(A, L, blk)
```

Comment:
I changed the name of this function to indicate that it was an internal utility.